phpseclib/Math/BigInteger/Engines/PHP64.php

```diff
@@ -192,7 +192,7 @@ public function extendedGCD(PHP64 $n)
      */
     public function gcd(PHP64 $n)
     {
-        extract($this->extendedGCD($n));
+        $gcd=$this->extendedGCD($n)['gcd'];
         return $gcd;
```

Comment:
I think I'd prefer just doing `return $this->extendedGCD($n)['gcd'];`. I can change that myself for both PHP32 / PHP64.